configuration/screensaver.md

```diff
@@ -1,28 +1,34 @@
 # Setting the screen saver/screen blanking
 
-## On the Console
+## On the console
 
-If you are using the Raspberry Pi solely on the console (no desktop GUI), you need to set the console blanking. The current setting, in seconds, can be displayed using
+By default, when running without a graphical desktop, Raspberry Pi OS will blank the screen after 10 minutes without user input: either mouse movement or keyboard input.
```

Comment:
Perhaps this sentence should be reworded to specifically mention Raspberry Pi OS Lite, and remove the confusing "by default" part?

Reply:
The "by default" was there already, but I agree that it is not required and just adds extra verbage. I've removed it.

Reply:
Still says "By default" in the "console" section of the page.

Reply:
Thanks - fixed.